patches/blockprotocol+0.0.1.patch

```diff
@@ -1,17 +1,45 @@
 diff --git a/node_modules/blockprotocol/core.d.ts b/node_modules/blockprotocol/core.d.ts
-index 205630a..1cf433a 100644
+index 205630a..0027afc 100644
 --- a/node_modules/blockprotocol/core.d.ts
 +++ b/node_modules/blockprotocol/core.d.ts
-@@ -66,6 +66,8 @@ export type BlockProtocolMultiSort = {
+@@ -25,18 +25,18 @@ export type BlockMetadata = {
+ };
+ 
+ export type BlockProtocolUpdateEntitiesAction<T> = {
++  accountId: string;
++  data: T;
++  entityId: string;
+   entityTypeId?: string | null;
+   entityTypeVersionId?: string | null;
+-  entityId: string;
+-  accountId?: string | null;
```

Comment:
this and other uses of `accountId` should remain optional in the BP types.

I would remove these changes since it will just be reordering them, and we can do that in the package itself.

Reply:
Done